sdk/communication/Azure.Communication.Identity/tests/CommunicationIdentityClient/CommunicationIdentityClientLiveTests.cs

```diff
@@ -21,12 +21,17 @@ namespace Azure.Communication.Identity.Tests
     /// </remarks>
     public class CommunicationIdentityClientLiveTests : CommunicationIdentityClientLiveTestBase
     {
+        /// <summary>
+        /// Options used to exchange an AAD access token of a Teams user for a new Communication Identity access token.
+        /// </summary>
+        internal GetTokenForTeamsUserOptions CTEOptions;
         /// <summary>
         /// Initializes a new instance of the <see cref="CommunicationIdentityClient"/> class.
         /// </summary>
         /// <param name="isAsync">A flag used by the Azure Core Test Framework to differentiate between tests for asynchronous and synchronous methods.</param>
         public CommunicationIdentityClientLiveTests(bool isAsync) : base(isAsync)
         {
+            CTEOptions = createTeamsUserParams().Result;
```

Comment:
Calling `.Result` is not a good practice. I'd avoid setting the CTEOptions in the constructor and rather used `[OneTimeSetup]` from [NUnit3](https://docs.nunit.org/articles/nunit/writing-tests/attributes/onetimesetup.html) which can be async.